listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -235,17 +235,23 @@ def fetch_listens_from_storage(self, user_name, from_ts, to_ts, limit, order, ti
             max_timestamp_window = -1
         else:
             max_timestamp_window = 432000 * time_range
-            if from_ts is not None:
+            if from_ts and to_ts:
+                pass
```

Comment:
we don't need an if statement which just has a `pass` here, this can be removed.